bucket/wireshark.json

```
@@ -31,6 +31,7 @@
     "suggest": {
         "nmap": "nmap"
     },
+    "post_install": "Write-Host -fore yellow \"*********************`nNote: Wireshark needs a packet sniffing library to capture packets from your network interface, but this must be installed separately. You can download NPcap at https://nmap.org/npcap/#download`n*********************\";",
```

Or just add npcap manifest and depends on it.
```suggestion
    "notes": [
        "Wireshark needs a packet sniffing library to capture packets from your network interface, but this must be installed separately",
        "You can download NPcap at https://nmap.org/npcap/#download",
    ],
```